### PR TITLE
Added whitespace before triple dots in template (Fixes #1729)

### LINKED ISSUE #1729: Add whitespace before triple dots in template
I like to do this because it's more beautiful.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
convinced ;-)

### CODE REVIEW

**Comment by normanjaeckel** [comment]
Ok.

File: openslides/agenda/static/templates/agenda/item-list.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h1 translate>Agenda</h1>
         <button os-perms-lite="agenda.can_manage" class="btn btn-default"
             ng-click="isDeleteMode = !isDeleteMode; uncheckAll()">
           <i class="fa fa-check-square-o"></i>
-          <translate>Select</translate>...
+          <translate>Select</translate> ...
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-list.html]: The dots should be inside the translation block. Maybe there is a language where they do not add the whitespace or use something else then the dots.

File: openslides/agenda/static/templates/agenda/item-list.html [hunk 2]
```diff
@@ -67,7 +67,7 @@ <h1 translate>Agenda</h1>
         </div>
         <button class="btn btn-default" ng-click="isFilter = !isFilter">
           <i class="fa fa-filter"></i>
-          <translate>Filter</translate>...
+          <translate>Filter</translate> ...
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-list.html]: Here the same

File: openslides/agenda/static/templates/agenda/item-list.html [hunk 3]
```diff
@@ -203,7 +203,7 @@ <h1 translate>Agenda</h1>
               <button type="submit" class="btn btn-primary" translate>
                 Update
               </button>
-              <a href="" ng-click="edit(item)" class="pull-right"><translate>Edit</translate>...</a>
+              <a href="" ng-click="editDialog(item)" class="pull-right"><translate>Edit</translate> ...</a>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-list.html]: Here the same

**Comment by ostcar** [comment]
There are a lot more places where the dots are not translated

**Comment by emanuelschuetze** [comment]
All done. Rebased. Ready for merge.

**Pull request merged**